### PR TITLE
[3/x] Add deployer Tx and UI for veBanny

### PR DESCRIPTION
**Pull request by jmilldotdev**
## What does this PR do and why?

Adds a section in the project tools drawer veNFT tab for launching a stock veBanny contract, and adds a transaction which handles the launch.

## Screenshots or screen recordings

image

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by jmilldotdev** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1654** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1654" target="_blank">Graphite</a> 
    * **PR #1655** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1655" target="_blank">Graphite</a> 
      * **PR #1656** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1656" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1656?utm_source=stack-comment).

File: src/hooks/veNft/transactor/LaunchVeNftTx.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    ) {
+      txOpts?.onDone?.()
+      return Promise.resolve(false)
+    }
+
+    return transactor(
+      contracts.JBVeNftDeployer,
+      'deployVeNFT',
+      [
+        projectId, // _projectId
```
Comment (tomquirk) [inline src/hooks/veNft/transactor/LaunchVeNftTx.ts:37]: **praise** ty for the arg comments!

**Review by tomquirk — CHANGES_REQUESTED** [review]
@jmilldotdev this LGTM. Will need to rebase the entire stack on `main` though; we shouldn't have `messages.po` files in PRs anymore <3

**Comment by tomquirk** [comment]
@jmilldotdev looks like some whacky merge conflicts in the translations files.

Give it a `git checkout main -- src/locales` and run extract again. Should be way easier in future, thanks for bearing with the current process <3

**Pull request merged**